frontend/src/shell/shell.css

```
@@ -0,0 +1,632 @@
+
+.cp-music-eq {
+  display: inline-flex;
+  align-items: flex-end;
+  gap: 2px;
+  height: 12px;
+  width: 14px;
+  justify-content: center;
+}
+.cp-music-eq > span {
+  width: 2px;
+  background: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)